webdriver/newsession/create-1.py

```diff
@@ -0,0 +1,24 @@
+#META: timeout=long
```

Comment:
IIRC the timeout is now set longer so it might be possible to merge these files.